lib/policies/proxy/proxy.js

```diff
@@ -77,7 +77,13 @@ module.exports = function (params, config) {
 
   const balancer = createStrategy(strategy, proxyOptions, urls);
 
-  const stripPathFn = params.stripPath ? (req) => { req.url = `${req.params[0]}${req._parsedUrl.search || ''}`; } : () => { };
+  const stripPathFn = params.stripPath
+    ? req => {
+      req.url = req.params[0]
+        ? `${req.params[0]}${req._parsedUrl.search || ''}`
+        : '';
+    }
+    : () => { };
```

Comment:
In case you're wondering how I came out with this formatting — I gave the code to Prettier.io and I pasted here the result :trollface: